packages/@aws-cdk/aws-iotevents/lib/event.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Specifies the actions to be performed when the condition evaluates to TRUE.
```

Comment:
I have to say, I don't understand what concept is this supposed to represent.

Where is the action that is performed when `condition` is `true` defined?

Reply:
[Step 3 of the roadmap](https://github.com/aws/aws-cdk/issues/17711) will allow us to set `actions`!
So now this document is indecipherable. As a matter of fact, condition does not mean anything now.
Hmm... Is it better that this PR includes implementation of `actions`? WDYT?

Reply:
How about we kick out `conditions` too for now from this PR? 😃 And then implement conditions and Actions together (they seem pretty related) in a separate PR?

Reply:
The detector model needed a reference to at least one AWS IoT Events input in order to deploy it. So if we kick out `conditions`, we also remove integration test for `DetectorModel` from this PR. No problem?

Reply:
Not sure I understand - `condition` is optional...?

Reply:
Yes. `condition` is optional for `Event`.
But the detector model must have at least one condition that has a reference to IoT Event Input.
![image](https://user-images.githubusercontent.com/11013683/146963656-ed5ead70-a57f-4702-bc9b-2e377bbab490.png)

- The `DetectorModel` has more than 1 `State`
- The `State` has more than 0 `Event`
- The `Event` has a `condition`, or not.
- And, the detector must have at least one condition that has a reference to IoT Event Input.

So I included events and `condition` in this PR, even though they are optional.

Reply:
Ok, thanks for the explanation! If that's the case, then we need to validate that this invariant (the `initialState` of the `DetectorModel` has at least one `Event` with a non-`undefined` `condition`) is satisfied in the `DetectorModel` class.

Reply:
@skinny85 
I've implemented the validation!

BTW, the first issue of this thread
> Where is the action that is performed when condition is true defined?

was resolved?

I should implement Action too in this PR? Or remove something feature from this PR?

Reply:
I'll take a look at the code, and I'll let you know 😉.